### PR TITLE
[Preprocessing] Migrate RandomContrast to KerasCV

### PR DESCRIPTION
**Pull request by ianstenbit**

### CODE REVIEW

**Review by divyashreepathihalli — APPROVED** [review]

**Comment by divyashreepathihalli** [comment]
Thank you for the PR Ian.
Can you please add a demo here - /keras-cv/examples/layers/preprocessing/classification
And post the demo output here?

File: keras_cv/layers/preprocessing/random_contrast.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            max = 1 + factor
+        self.factor = preprocessing.parse_factor((min, max), min_value=-1, max_value=2)
+        self.seed = seed
+
+    def get_random_transformation(self, **kwargs):
+        return self.factor()
+
+    def augment_image(self, image, transformation, **kwargs):
+        contrast_factor = transformation
+        output = tf.image.adjust_contrast(image, contrast_factor=contrast_factor)
```
Comment (LukeWood) [inline keras_cv/layers/preprocessing/random_contrast.py:72]: does this need to consider `value_range`?

Comment (ianstenbit) [reply keras_cv/layers/preprocessing/random_contrast.py:72]: No -- tf.image.adjust_contrast doesn't have a required value range (and the underlying Keras layer didn't make any assumptions about input range)

**Review by divyashreepathihalli — APPROVED** [review]

**Review by LukeWood — APPROVED** [review]
Looks good to me

**Comment by ianstenbit** [comment]
Since our GCB integration is currently failing, I've run all affected tests manually on a GCP GPU VM, and they're all green, so I will go ahead and merge.

**Pull request merged**